### ISSUE TITLE
Перебои работы Интернета в Первоуральск (Свердловская область (йота, мегафон) [03:00 — 04:00 24.12.2019]

### ISSUE THREAD

**Issue by point807** [podivilov/runet-blackout#31]
У некоторых пользователей йоты и мегафон было полностью недоступно интернет подключение, также наблюдалось временное потери связи у части пользователей йоты

**Comment by point807** [comment]
Upd. C 11:00-12:00 были потери скорости передачи данных

**Comment by point807** [comment]
Upd. Мегафон прислало СМС с извинениями за проблемы 24 числа и дали в подарок неделю безлимитных звонков
